uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/federatedlogin/uilocators/GeSsoIDPlogin.java

```diff
@@ -0,0 +1,36 @@
+package org.cloudfoundry.identity.uaa.integration.feature.federatedlogin.uilocators;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class GeSsoIDPlogin {
```

Comment:
GeSsoIDPlogin is a confusing class name.  You are not using GE SSO for testing. Please think of a better name. For example, IdpLoginUI or IdpLoginScreen